modules/op-release-notes-1-9.adoc

```diff
@@ -0,0 +1,240 @@
+// Module included in the following assembly:
+//
+// * cicd/pipelines/op-release-notes.adoc
+:_content-type: REFERENCE
+[id="op-release-notes-1-9_{context}"]
+= Release notes for {pipelines-title} General Availability 1.9
+
+With this update, {pipelines-title} General Availability (GA) 1.9 is available on {product-title} 4.11 and later versions.
+
+[id="new-features-1-9_{context}"]
+== New features
+
+In addition to the fixes and stability improvements, the following sections highlight what is new in {pipelines-title} 1.9.
+
+[id="pipelines-new-features-1-9_{context}"]
+=== Pipelines
+
+* With this update, you can specify pipelines parameters and results in arrays and object dictionary forms.
+
+* This update provides support for Container Storage Interface (CSI) and projected volume for your workspace.
+
+* This update supports the matrix feature to organize tasks in a pipeline. For example, you can define a single task in your pipeline which executes multiple times based on the matrix parameters. This update is part of an alpha preview feature.
+
+* With this update, you can specify the `stdoutConfig` and `stderrConfig` parameters when defining pipeline steps. Defining these parameters helps to capture standard output and standard error associated with steps to local files.
+
+* With this update, you can add variables in the `steps[].onError` event handler, for example, `$(params.CONTINUE)`.
+
+* With this update, you can use the output from the `finally` task in the `PipelineResults` definition. For example, `$(finally.<pipelinetask-name>.result.<result-name>)`, where `<pipelinetask-name>` denotes the pipeline task name and `<result-name>` denotes the result name.
+
+* This update supports task-level resource requirements for a task run.
+
+* With this update, you do not need to recreate parameters that are shared based on their name between a pipeline and the defined tasks. This update is part of an alpha preview feature.
```

Comment:
```suggestion
* With this update, you do not need to recreate parameters that are shared, based on their names, between a pipeline and the defined tasks. This update is part of an alpha preview feature.
```

Reply:
or move the "based on their names" to after the "between a pipeline and the defined tasks"